chainer/functions/connection/convolution_nd.py

```diff
@@ -421,3 +421,254 @@ def convolution_nd(x, W, b=None, stride=1, pad=0, cover_all=False):
         return func(x, W)
     else:
         return func(x, W, b)
+
+
+def convolution_1d(x, W, b=None, stride=1, pad=0, cover_all=False):
+    """1-dimensional convolution function.
```

Comment:
IMO we don't need full documentation for aliases, because it's difficult to maintain.
How about simply redirecting to `convolution_nd`?

Reply:
OK, let's do so.